code/modules/mob/living/simple_animal/hostile/terror_spiders/objects.dm

```diff
@@ -0,0 +1,353 @@
+		var/mob/living/L = target
+		if(L.reagents)
+			L.reagents.add_reagent("ketamine",30)
+		//options:
+		//               terror_white_tranq, 0.1 metabolism, paralysis, cycle >= 10
+		//               sodium_thiopental, 0.7 metabolism, paralysis, cycle >= 5
+		//               ketamine, 0.8 metabolism, paralysis, cycle >= 10
+		//               Coniine, 0.05 metabolism, rapid respitory failure
+
+
+/obj/effect/spider/terrorweb
+	name = "terror web"
+	desc = "it's stringy and sticky"
+	icon = 'icons/effects/effects.dmi'
+	anchored = 1 // prevents people dragging it
+	density = 0 // prevents it blocking all movement
+	health = 20 // two welders, or one laser shot (15 for the normal spider webs)
+	icon_state = "stickyweb1"
+
+
+/obj/effect/spider/terrorweb/New()
+	if(prob(50))
+		icon_state = "stickyweb2"
+
+/obj/effect/spider/terrorweb/proc/DeCloakNearby()
+	for(var/mob/living/simple_animal/hostile/poison/terror_spider/gray/G in view(6,src))
+		G.GrayDeCloak()
+		G.Aggro()
+
+/obj/effect/spider/terrorweb/CanPass(atom/movable/mover, turf/target, height=0, air_group=0)
+	if(air_group || (height==0)) return 1
+	if(istype(mover, /mob/living/simple_animal/hostile/poison/terror_spider))
+		return 1
+	else if (istype(mover, /obj/item/projectile/terrorqueenspit))
+		return 1
+	else if (istype(mover, /obj/item/projectile/terrorempressspit))
+		return 1
+	else if(istype(mover, /mob/living))
+		if(prob(80))
+			to_chat(mover, "<span class='danger'>You get stuck in \the [src] for a moment.</span>")
+			var/mob/living/M = mover
+			M.Stun(5) // 5 seconds.
+			M.Weaken(5) // 5 seconds.
+			M.emote("scream")
+			DeCloakNearby()
+			return 1
+	else if(istype(mover, /obj/item/projectile))
+		return prob(20)
+	return 1
+
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: LOOT DROPS -----------------------------------
+// --------------------------------------------------------------------------------
+
+
+/obj/item/clothing/suit/armor/terrorspider_carapace
+	name = "spider carapace armor"
+	desc = "A carved section of terror spider carapace that can be used as crude body armor."
+	icon_state = "armor-combat"
+	item_state = "bulletproof"
+	//DEFAULT VALUES: armor = list(melee = 50, bullet = 15, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)
+	armor = list(melee = 50, bullet = 30, laser = 25, energy = 10, bomb = 25, bio = 0, rad = 0)
+	// our armor half as effective against lasers, but 2x as effective against bullets. Makes sense - terror spiders had to fight syndicate first, only recently encountered NT
+
+
+// venom glands.
+
+/obj/item/weapon/reagent_containers/terrorspider_parts
+	icon = 'icons/mob/terrorspider.dmi'
+	desc = "A body part from a terror spider."
+
+/obj/item/weapon/reagent_containers/terrorspider_parts/toxgland_green
+	name = "green terror spider venom gland"
+	icon_state = "ts_venomgland"
+	origin_tech = "biotech=5;materials=2;combat=4"
+	New()
```

Comment:
this might be okay in the reagent container file
it's not when it's in it's own file